types/openui5/sap.ui.integration.d.ts

```diff
@@ -580,66 +599,156 @@ declare module "sap/ui/integration/ActionDefinition" {
   }
 }
 
-declare module "sap/ui/integration/Designtime" {
+declare module "sap/ui/integration/widgets/Card" {
   import {
-    default as ManagedObject,
-    $ManagedObjectSettings,
-  } from "sap/ui/base/ManagedObject";
+    CardBlockingMessageType,
+    CardActionType,
+    CardDataMode,
+    CardDesign,
+    CardPreviewMode,
+    CardArea,
+  } from "sap/ui/integration/library";
 
-  import { CardFacade } from "sap/ui/integration/widgets/Card";
+  import IllustratedMessageType from "sap/m/IllustratedMessageType";
 
-  import ManagedObjectMetadata from "sap/ui/base/ManagedObjectMetadata";
+  import IllustratedMessageSize from "sap/m/IllustratedMessageSize";
+
+  import { default as CardBase, $CardBaseSettings } from "sap/f/CardBase";
+
+  import ActionDefinition from "sap/ui/integration/ActionDefinition";
+
+  import Event from "sap/ui/base/Event";
+
+  import Control from "sap/ui/core/Control";
+
+  import { URI, ID, MessageType } from "sap/ui/core/library";
+
+  import Host from "sap/ui/integration/Host";
+
+  import ElementMetadata from "sap/ui/core/ElementMetadata";
+
+  import {
+    PropertyBindingInfo,
+    AggregationBindingInfo,
+  } from "sap/ui/base/ManagedObject";
 
   /**
-   * @SINCE 1.75
-   * @EXPERIMENTAL (since 1.75)
+   * @EXPERIMENTAL (since 1.114)
    *
-   * Brings JavaScript capabilities for an {@link sap.ui.integration.widgets.Card} where custom logic can
-   * be implemented.
+   * Settings for blocking message that ocurred in a {@link sap.ui.integration.widgets.Card}
    */
-  export default class Designtime extends ManagedObject {
+  export type BlockingMessageSettings = {
     /**
-     * Constructor for a new `Designtime`.
+     * Blocking message type
+     */
+    type: CardBlockingMessageType | keyof typeof CardBlockingMessageType;
+    /**
+     * Illustration type
+     */
+    illustrationType: IllustratedMessageType;
+    /**
+     * Illustration size
+     */
+    illustrationSize?:
+      | IllustratedMessageSize
+      | keyof typeof IllustratedMessageSize;
+    /**
+     * Title
+     */
+    title: string;
+    /**
+     * Description
+     */
+    description?: string;
+    /**
+     * Response object in case of a network error
+     */
+    httpResponse?: Response;
+  };
+
+  /**
+   * @SINCE 1.62
+   *
+   * A control that represents a container with a header and content.
+   *
+   * Overview: Cards are small user interface elements which provide the most important information from an
+   * app, related to a specific role or task. The information is represented in a compact manner, allowing
+   * for actions to be executed. Cards can be described as small representations of an app which can be integrated
+   * in different systems.
+   *
+   * The integration card is defined in a declarative way, using a manifest.json to be:
+   * 	 - Easily integrated into apps
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/openui5/sap.ui.integration.d.ts;
  Line: 680;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.